--- ../.superpowers/sdd/2026-09-13-apple-signin/integrated/browser/apple-png-after.sha256	2026-09-13 02:33:28
+++ ../.superpowers/sdd/2026-09-13-apple-signin/integrated/browser/apple-png-post-fix.sha256	2026-09-13 02:40:17
@@ -1,7 +1,7 @@
 02b66af9c07877db007719d93f279b961324e5e954a86f377622016233b32310  ../docs/screenshots/apple-signin-welcome-landscape.png
 1668352ac178d111bd67794b722d4f64bc08007c7cbe9556ef89195ac802f300  ../docs/screenshots/apple-create-account.png
 184e1aa600ff02e0a85500aeb9a098a1c73d57735363159efca5e3870ca6cb83  ../docs/screenshots/apple-signin-welcome.png
+1cbe6b87eb12b773d703c229f6fd3ec5c970c979215bcc4f855e8c6e8048122e  ../docs/screenshots/apple-link-result-uncertain.png
 9a966e88a26ae1767ec1c98dcebaf14e321ab9470eba5abd37aea08a496646dc  ../docs/screenshots/apple-signin-methods.png
-be923d513e6a3d1880a94bc0f8ebae8a8e58916f3160a7cbcba4479de36d7659  ../docs/screenshots/apple-link-result-uncertain-landscape.png
-e5d55d92836d150c2205c02cdce3febeedbfb857299ec9e5c73d39898bfcec86  ../docs/screenshots/apple-link-result-uncertain.png
-f8b41d5ed0813bc8d973bebab58c9a7863b9505979ed225c96e484decc939330  ../docs/screenshots/apple-link-confirm-landscape.png
+e1a8ac9df0a2f72cbed23929cefd01e723871a82ecef0081337b62b712ac4149  ../docs/screenshots/apple-link-confirm-landscape.png
+f98447bef25ad9acbbb64a54b49a1264e6da6c9fd33148787b7d4f81b28e288c  ../docs/screenshots/apple-link-result-uncertain-landscape.png
